sklearn/impute/_iterative.py

```diff
@@ -144,6 +144,12 @@ class IterativeImputer(_BaseImputer):
         the missing indicator even if there are missing values at
         transform/test time.
 
+    keep_empty_features : bool, default=False
+        If True, features that consist exclusively of missing values `fit` is
+        called not removed when calling `transform`. The imputed value is `0`.
```

Comment:
Ditto: I think there is a small syntax mistake. Also, is using this affirmative phrasing better?
```suggestion
        If True, features that consist exclusively of missing values when
        `fit` is called are returned in results when `transform` is called.
        The imputed value is always `0` except when
        `initial_strategy="constant"` in which case `fill_value` will be
        used instead.
```